aggregate-repositories.sh

```diff
@@ -29,6 +29,11 @@ do
         fi
     done
 
+    # add special use case for sda.md links
+
+    sed -i -E 's/cmd\/(.+)\//services\//g' docs/services/sda.md
```

Comment:
```suggestion
    sed -i -E 's#cmd\/(.+)\/#''#g' docs/services/sda.md
```
Unless I am missing something, all `md` files live under the same  folder. I also changed the separator to `#` for more readability but that's a matter of taste.